tests/__init__.py

```diff
@@ -86,3 +95,62 @@ def __init__(self, cubes, filename, attributes, settings=None, **kwargs):
         self.copy_provenance = mock.Mock(return_value=self)
 
     group = PreprocessorFileBase.group
+
+
+def create_realistic_4d_cube():
```

Comment:
It would be more in line with the pytest style of using fixtures to define this is a fixture in conftest.py